aeron-client/src/main/c/util/aeron_fileutil.c

```diff
@@ -364,9 +370,8 @@ int aeron_map_new_file(aeron_mapped_file_t *mapped_file, const char *path, bool
         return -1;
     }
 
-    if (0 != aeron_mmap(mapped_file, fd, fill_with_zeroes))
+    if (0 != aeron_mmap(mapped_file, fd, fill_with_zeroes, path))
```

Comment:
There is no need to pass the path in here for the purposes of logging.  The better approach is to use `AERON_APPEND_ERR`, e.g:

```java
if (0 != aeron_mmap(mapped_file, fd, fill_with_zeroes))
{
    AERON_APPEND_ERR("path=%s", path);
    close(fd);
    if (-1 == remove(path))
    {
```
This is the main purpose of `AERON_APPEND_ERR`, we shouldn't need to pass values up/down the call stack just to log them.  We can append context to the error stack at the point where we have the information.  This also provides additional trace information (file/line number) so that we can see the path through the stack where the error occurred.  Even if there is no additional context information to include in the log, we should do `AERON_APPEND_ERR("%", "")` anyway, so that we capture the trace information.

Reply:
Makes sense. I'll update the code accordingly.

Reply:
Done.